packages/@guardian/source-foundations/src/typography/index.ts

```diff
@@ -82,14 +82,22 @@ const titlepiece = Object.fromEntries(
  *
  * `headline.xlarge()` -> 50px
```

Comment:
Given that this docstring (and the docstrings for other `TypographyApi` instances) is here purely for IntelliSense, I think they're no longer necessary. What do you think?

![Screenshot 2021-12-08 at 07 55 00](https://user-images.githubusercontent.com/5931528/145170131-805b56b9-b6f0-4d46-ad99-163e5241b205.png)

Reply:
Removed in bbeb4f69c07586cbcade3cd2587e81b8f5e8627e